runtime/sema/type.go

```diff
@@ -5261,6 +5347,19 @@ func checkSubTypeWithoutEquality(subType Type, superType Type) bool {
 			//
 			// The holder of the reference may not gain more permissions or knowledge.
 
+			return false
+
+		case *InterfaceType:
+			switch typedInnerSubType := typedSubType.Type.(type) {
+			case *CompositeType:
```

Comment:
Please also add a comment like below. 

This seems to implement `&T <: &U`, where `T` is a concrete type and `U` is an interface type.
This is similar to the rule `&T <: &{U}`. 

Maybe point that out and include a comment similar to the one for that rule.